### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 7

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#307] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322086-0e92bc3d-a9f2-43b2-911f-cf91e6b0716e.png)

**Comment by Exyss** [comment]
Problema e soluzione analoghi all'Esercizio 1 (#102)

**Comment by LuginiAndrea** [comment]
Riformuliamo il problema.
Costruiamo l'insieme $$T = \bigcup_{i = 0}^{\infty} \left(12 + 4i, 13 + 4i, 14 + 4i, 15 + 4i\right)$$ (Leggete l'uso delle tonde come le graffe degli insiemi)
$$\forall n \geq 15, n + 1 = 12 + \left(Q\left(n+1,4\right)-3\right) \cdot 4 + R\left(n+1,4\right) \textrm{(Q è il quoziente fra interi, inoltre togliamo 3 perchè sommiamo già 12 = $4 \cdot 3$, R è il resto del quoziente fra interi)}$$
$$R\left(n+1,4\right) \in \left[0, 3\right], Q\left(n+1,4\right)-12 \in \left[1, + \infty\right)$$
Facile notare come $$12 + R\left(n+1, 4\right) \in \left[0 + 12, 3 + 12\right] = \left[12, 15\right] = x$$
Inoltre possiamo sostituire $Q\left(n+1,4\right) - 12$ con $i \in \left[i, +\infty\right)$$
Possiamo riscrivere la somma come $x + 4i$
Per casi:

1. $12 + 4i$
2. $13 + 4i$
3. $14 + 4i$
4. $15 + 4i$

Facile notare come, per costruzione, ognuno di questi casi ricada in $T_i$
